app/assets/v2/js/pages/new_bounty.js

```diff
@@ -462,6 +462,15 @@ $('#sync-issue').on('click', function(event) {
 });
 
 $('#issueURL').focusout(function() {
+  for (var i = 0; i <= document.blocked_urls.length; i++) {
```

Comment:
let / const

Reply:
do we wanna add a ` document.blocked_urls` check  ?

Reply:
> let / const

i increments; why would i want to make it a constant?

> document.blocked_urls

no.. itll always exist

Reply:
> i increments; why would i want to make it a constant?

aka `let` to avoid hoisting it  :P

Reply:
TIL about hoisting.. committing a few now